app/oauth.py

```diff
@@ -1,12 +1,14 @@
 import json
 import os
 import re
-
 import requests
-from django.contrib.auth.models import User as DefaultUser
+
 from dotenv import load_dotenv
 from oauth2_provider.models import AccessToken
 
+from django.contrib.auth.models import User as DefaultUser
```

Comment:
I think this should remain about the dotenv import

Reply:
The PEP8 styling guide says something else, I had it above dotenv before i reordered it because of the previous comment Kosy left alongside the style guide link.